### DISCUSSION TITLE
doesnt wurk

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1079] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
blobwifi, caub, and other wifi bypassers. DO NOT WORK. (my school iz so smartt)

**Comment by ohiobombs** [comment]
caub works, so do you mean cauDNS?

**Comment by Blobby-Boi** [comment]
wdym it doesnt work? does the onc file not import?

**Reply by ghost** [reply]
it imports, but it does nothing.

**Reply by CGRmags** [reply, marked as answer]
try  mporting at home it worked for me bc school can somehow block it from the wi-fi
